src/e2e-test/e2e-test.spec.js

```diff
@@ -0,0 +1,76 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+const supertest = require('supertest');
+const express = require('express');
+var fs = require("fs");
+var path  = require('path');
+var constants = require('../lib/constants/constants');
+const API_KEY_HEADER = "X-MS-CONVERSION-API-KEY";
+const apiKeys = ['123', '456'];
```

Comment:
Why do we need two api keys?

Reply:
Just for example, we can support multiple keys